### PR TITLE
Fixing style and typos

### PR DESCRIPTION
**Pull request by mari-linhares**

### CODE REVIEW

File: tutorials/image/cifar10_estimator/cifar10_main.py [hunk 1]
```diff
@@ -317,9 +317,11 @@ def _resnet_model_fn(features, labels, mode):
...
         for x in np.array([82, 123, 300], dtype=np.int64)
     ]
-    staged_lr = [FLAGS.learning_rate * x 
-        for x in [1, 0.1, 0.01, 0.002]]
-    
+    staged_lr = [
+        FLAGS.learning_rate * x
+        for x in [1, 0.1, 0.01, 0.002]
+    ]
+
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/cifar10_main.py]: This should all fit on a single line.

File: tutorials/image/cifar10_estimator/cifar10_model.py [hunk 2]
```diff
@@ -17,9 +17,8 @@
 from __future__ import division
 from __future__ import print_function
 
-import tensorflow as tf
-
 import model_base
+import tensorflow as tf
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/cifar10_model.py]: This was correct the way it was before. You want to import libraries before you import your own files.

File: tutorials/image/cifar10_estimator/generate_cifar10_tfrecords.py [hunk 3]
```diff
@@ -81,6 +80,7 @@ def convert_to_tfrecord(input_files, output_file):
       record_writer.write(example.SerializeToString())
   record_writer.close()
 
+
 def main(argv):
   del argv  # Unused.
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/generate_cifar10_tfrecords.py]: Typically I've seen this argument called `unused_argv` without any lines that use `del`.

File: tutorials/image/cifar10_estimator/cifar10_model.py (hunk 2 above)
Comment (mari-linhares) [inline tutorials/image/cifar10_estimator/cifar10_model.py]: Funny, pylint corrected me, will undo this.

File: tutorials/image/cifar10_estimator/cifar10_main.py [hunk 4]
```diff
@@ -317,9 +317,10 @@ def _resnet_model_fn(features, labels, mode):
         num_batches_per_epoch * x
         for x in np.array([82, 123, 300], dtype=np.int64)
     ]
-    staged_lr = [FLAGS.learning_rate * x 
-        for x in [1, 0.1, 0.01, 0.002]]
-    
+    staged_lr = [
+        FLAGS.learning_rate * x for x in [1, 0.1, 0.01, 0.002]
+    ]
+
```
Comment (nealwu) [inline tutorials/image/cifar10_estimator/cifar10_main.py]: You can just make the whole thing one line like this:

`staged_lr = [FLAGS.learning_rate * x for x in [1, 0.1, 0.01, 0.002]]`

**Comment by nealwu** [comment]
Looks good!

**Pull request merged**